tests/parsers/test_flawfinder_parser.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+#
+#   Copyright 2017 David Carlos  <ddavidcarlos1392@gmail.com>
+#   This library is free software; you can redistribute it and/or
+#   modify it under the terms of the GNU Lesser General Public
+#   License as published by the Free Software Foundation; either
+#   version 2.1 of the License, or (at your option) any later version.
+#
+#   This library is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#   Lesser General Public License for more details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+#   License along with this library; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+#   USA
+
+import os
+import unittest
+
+from firehose.parsers.flawfinder  import load_file,  parse_file
+from firehose.model import Analysis, Issue, Sut, Trace
+
+class TestParseXml(unittest.TestCase):
+    def parse_example(self, filename):
+        report = load_file(os.path.join(os.path.dirname(__file__),
```

Comment:
Given that it's a file-like object, maybe turn into:

    path = os.path.join(...)
    with open(path) as f:
        return parse_file (f)